sagemaker-triton/fil_ensemble/2_triton_xgb_fil_ensemble.ipynb

```diff
@@ -0,0 +1,908 @@
+{
```

Comment:
<p>Avoid "here" link text for accessibility - recommended <code>the instructions for <a href="link.html" target="_blank">Packaging the Conda Environment</a></code></p><p> </p><p>Please apply to all occurrences in this PR.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3524/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Corrected